webpack.config.js

```diff
@@ -15,8 +15,10 @@ const pjson = require("./package.json");
 
 const ENV = process.env.ENV == null ? "development" : process.env.ENV;
 const NODE_ENV = process.env.NODE_ENV == null ? "development" : process.env.NODE_ENV;
+const SELF_HOSTED = ENV === "selfhosted" || process.env.SELF_HOSTED === "true";
 
 const envConfig = config.load(ENV);
+envConfig.dev = SELF_HOSTED ? envConfig.dev.selfHosted : envConfig.dev.cloud;
```

Comment:
We have a config file for self hosted which we could put this in. We need to be careful though since the ports don't match what users would typically use. Unless we update all self-hosted instances to use the new ports which would let us simplify things?

Reply:
I wonder if we should just bite the bullet and add support for `env.development.json`.